src/osip-monitoring-view/src/main/java/edu/kit/pse/osip/monitoring/view/dialogs/HelpDialog.java

```diff
@@ -1,7 +1,136 @@
 package edu.kit.pse.osip.monitoring.view.dialogs;
 
+import edu.kit.pse.osip.core.utils.language.Translator;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.Tab;
+import javafx.scene.control.TabPane;
+import javafx.scene.image.Image;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.text.Font;
+import javafx.scene.text.Text;
+import javafx.scene.text.TextFlow;
+import javafx.stage.Stage;
+
 /**
  * This window shows a short description of OSIP and how to use it.
+ *
+ * @author David Kahles
+ * @version 1.0
  */
-public class HelpDialog extends javafx.stage.Stage {
+public class HelpDialog extends Stage {
```

Comment:
Das Problem ist, JavaFX besitzt eine feste Größe für alle Geräte, und bei mir ist die Standard-Schriftgröße zu klein, weswegen ich es erhöht habe. Falls du mal in meinen Code schaust, wirst du feststellen, dass ich die Schriftgröße aus der Bildschirmhöhe berechne. Da das außerdem eine Konstante ist, kann man dies anpassen.